mir-module/src/main/resources/xsl/metadata/mir-metadata-box.xsl

```diff
@@ -183,7 +183,13 @@
               mode="present" />
             <xsl:apply-templates select="mycoreobject/metadata/def.modsContainer/modsContainer/mods:mods/mods:originInfo[@eventType='update']/mods:dateModified"
               mode="present" />
-            <xsl:apply-templates mode="present" select="mycoreobject/metadata/def.modsContainer/modsContainer/mods:mods/mods:identifier[@type!='open-aire' and @type!='intern' and @type!='issn']" />
+
+            <xsl:variable name="identifierTypes" select="document('classification:metadata:-1:children:identifier')/mycoreclass/categories" />
+            <xsl:for-each select="$identifierTypes/category[@ID!='open-aire' and @ID!='intern' and @ID!='issn']">
```

Comment:
This only works with a "flat" identifier classification, should be `$identifierTypes//category[...]` in order to be more robust.